spec/services/claims/fee_calculator/unit_price_spec.rb

```diff
@@ -57,129 +57,172 @@
-      let(:case_type) { create(:case_type, :elected_cases_not_proceeded) }
-      let(:fee_type) { create(:fixed_fee_type, :fxenp) }
-      let(:fee) { create(:fixed_fee, fee_type: fee_type, claim: claim, quantity: 1) }
+      let(:params) do
+        {
+          format: :json,
+          id: claim.id,
+          advocate_category: 'Junior alone',
+          fee_type_id: fee.fee_type.id,
+          fees: {
+            "0": { fee_type_id: fee.fee_type.id, quantity: fee.quantity }
+          }
+        }
+      end
 
-      it_returns 'a successful fee calculator response'
-      it_returns 'a fee calculator response with amount', amount: 194.0
-    end
+      context 'for a case-type-specific fixed fee' do
+        it_returns 'a successful fee calculator response'
+        it_returns 'a fee calculator response with amount', amount: 130.0
+      end
 
-    context 'for a non-case-type-specific fixed fee (standard appearance fee/adjournments)' do
-      let(:saf_fee) { create(:fixed_fee, :fxsaf_fee, claim: claim, quantity: 1) }
+      context 'for a case-type-specific fixed fee with fixed amount (elected case not proceeded)' do
+        let(:case_type) { create(:case_type, :elected_cases_not_proceeded) }
+        let(:fee_type) { create(:fixed_fee_type, :fxenp) }
+        let(:fee) { create(:fixed_fee, fee_type: fee_type, claim: claim, quantity: 1) }
 
-      before do
-        params.merge!(fee_type_id: saf_fee.fee_type.id)
-        params[:fees].merge!("1": { fee_type_id: saf_fee.fee_type.id, quantity: saf_fee.quantity })
+        it_returns 'a successful fee calculator response'
+        it_returns 'a fee calculator response with amount', amount: 194.0
       end
 
-      it_returns 'a successful fee calculator response'
-      it_returns 'a fee calculator response with amount', amount: 87.0
-    end
+      context 'for a non-case-type-specific fixed fee (standard appearance fee/adjournments)' do
+        let(:saf_fee) { create(:fixed_fee, :fxsaf_fee, claim: claim, quantity: 1) }
 
-    # TODO: deprecated fee type - to be removed
-    context 'for a case-type-specific fixed fee case uplift' do
-      let(:uplift_fee) { create(:fixed_fee, :fxacu_fee, claim: claim, quantity: 1) }
+        before do
+          params.merge!(fee_type_id: saf_fee.fee_type.id)
+          params[:fees].merge!("1": { fee_type_id: saf_fee.fee_type.id, quantity: saf_fee.quantity })
+        end
 
-      before do
-        params.merge!(fee_type_id: uplift_fee.fee_type.id)
-        params[:fees].merge!("1": { fee_type_id: uplift_fee.fee_type.id, quantity: uplift_fee.quantity })
+        it_returns 'a successful fee calculator response'
+        it_returns 'a fee calculator response with amount', amount: 87.0
       end
 
-      it_returns 'a successful fee calculator response'
-      it_returns 'a fee calculator response with amount', amount: 26.0
-    end
+      # TODO: deprecated fee type - to be removed
```

Comment:
Can it be removed now? Or soon?

Reply:
This only appears in the PR because i indented the AGFS claims context. The case type specific fixed fee uplifts themselves still exist (seeds etc). Therefore i'd say that the test should be removed inline with the actual data (in case we need to add the uplifts backin - unlikely)

Will create a tech debt ticket for this